testsuite/src/main/java/io/netty5/testsuite/transport/socket/SocketCancelWriteTest.java

```diff
@@ -161,7 +165,17 @@ public void testCancelWrite(ServerBootstrap sb, Bootstrap cb) throws Throwable {
             throw ch.exception.get();
         }
         assertEquals(0, ch.counter.get());
+        assertNull(ch.received);
         assertEquals(preferredAllocator().copyOf(new byte[] { 'b', 'c', 'e' }), sh.received);
+        releaseReceivedBuffer(sh.received);
+    }
+
+    private void releaseReceivedBuffer(Object buffer) {
```

Comment:
You can use `Resource.dispose()` which does the same thing.

Reply:
Thanks, fixed.